apps/storybook/src/stories/datacom-list.stories.tsx

```diff
@@ -18,58 +16,196 @@ export default {
     },
     type: {
       name: 'Type',
-      control: 'select',
-      defaultValue: 'numbers',
       description: 'List type  is default  numbers if not set',
       options: ['numbers', 'lowercase', 'upercase', 'roman'],
     },
+    heading: {
+      name: 'Heading',
+      description: 'For heading style of list inside paragrap',
```

Comment:
Misspelt 'paragrap'. Should be 'paragraph'.

Reply:
Fixed.